src/constants/examples.constant.ts

```diff
@@ -2,7 +2,19 @@ export const EXAMPLES = {
   DEAL_ID: '0030000321',
   FACILITY_ID: '0030000322',
   PARTY_ID: '00001234',
+  BUNDLE_ID: '0000233092',
   CURRENCY: 'USD',
   DEAL_OR_FACILITY_VALUE: 501927.25,
   COUNTRY_CODE: 'GBR',
+  BUNDLE_STATUS_CODE: '10',
+  BUNDLE_STATUS_DESC: 'Pending Input With Drafts',
+  DATE_ONLY_STRING: '2023-04-29',
```

Comment:
Do we really need this? Swagger UI generates a suitable example automatically based on the format we specify

Reply:
Good point, removed.